cibyl/utils/fs.py

```diff
@@ -0,0 +1,129 @@
+"""
```

Comment:
Is this file an exact copy of the tripleo one or is it modified? Any reason to not importing from there?

Reply:
I want to keep cibyl and tripleo independent from one another. I have though of having a 'utils' folder common to both, but such a thing I just copied the file over and that is it.

Reply:
Yes, I can see how it might be problematic, we could easily end up with circular dependencies. I agree, just for this module is fine. We can revisit in the future if more code needs to be shared.